src/main/flight/pid_luxfloat.c

```diff
@@ -80,22 +84,24 @@ STATIC_UNIT_TESTED int16_t pidLuxFloatCore(int axis, const pidProfile_t *pidProf
 
     SET_PID_LUX_FLOAT_CORE_LOCALS(axis);
 
+    pidInitFilters(pidProfile);
```

Comment:
Yes, we need PID init function. There never seems to be time to make one though...